src/components/notification/notification.tsx

```diff
@@ -0,0 +1,59 @@
+import { CSSProperties, FC, ReactElement, ReactNode, useEffect } from 'react';
+import { NotificationProps } from '@components/notification/types';
+import styles from './notification.module.scss';
+import classNames from 'classnames/bind';
+import { CloseIcon, ErrorIcon, InfoIcon, SuccessIcon } from '@/components';
+
+const cx = classNames.bind(styles);
+export const Notification: FC<NotificationProps> = ({
+  uid,
+  title,
+  onClose,
+  icon = null,
+  type = 'info',
+  duration = 4000,
+  description,
+  className,
+}): ReactElement => {
+  const adjustedDuration = type === 'error' ? 7000 : duration;
```

Comment:
1) 7000 to put in a file with constants
2) error, info, success - create an enum in a file with constants